src/style.css

```diff
@@ -0,0 +1,19 @@
+.zeromgn {
+  margin: 0;
+}
+
+.ht {
+  height: 100px;
+}
+
+.cred {
+  background-color: red;
+}
+
+.cgreen {
+  background-color: green;
+}
+
+.cblue {
+  background-color: blue;
+}
```

Comment:
class names are very bad, I don't understand your shortened names, the same will happen if any other developer decides to look up something in your code. You need to give specific and clear names